Sketcher/OffsetInSketcher.FCMacro

```diff
@@ -0,0 +1,431 @@
+__name__   = "OffsetInSketcher"
+__author__  = "edi"
+__version__ = "0.03"
+__License__ = 'License identifier from https://spdx.org/licenses/, e.g. LGPL-2.0-or-later as FreeCAD, MIT, CC0-1.0'
+__Help__ = 'Select some lines in a sketch. Start the macro. Show offset value using the rubber band.'
+__Status__ = 'Stable'
+__Requires__ = 'e.g. FreeCAD >= v0.18'
+__date__    = "17/12/2020"
+
+from PySide import QtGui
+from pivy.coin import *
+if App.Version()[1].find('18') == 0: import DraftTrackers, Draft
+if App.Version()[1].find('19') == 0: import draftguitools.gui_trackers as DraftTrackers
+import re, math
+#
+#  create an offset line in sketcher
+#
+# - select a chain of lines and arcs in an opened sketch
+# - show value of offset distance and appearance of offset lines using a "rubber band"
+#
+class OffsetDialog:
+#
+# read selected items and show the offset distance
+#
+	def __init__(self):
+		self.view = FreeCADGui.ActiveDocument.ActiveView 
+		self.OffsetLine = [] 
```

Comment:
If you agree to make an effort that doesn't bring anything to your code but solely make it closer to Python style "conventions", https://www.python.org/dev/peps/pep-0008/ explains widely-accepted rules.